rllib/utils/pre_checks/env.py

```diff
@@ -21,12 +25,38 @@ def check_env(env: EnvType) -> None:
         ValueError: If env is not an instance of SUPPORTED_ENVIRONMENT_TYPES.
         ValueError: See check_gym_env docstring for details.
     """
-    from ray.rllib.env import BaseEnv, MultiAgentEnv, RemoteBaseEnv, VectorEnv
+    from ray.rllib.env import (
```

Comment:
Now that `disable_env_checking` is False by default in the Trainer config, we need to add a message here on how the user can disable the checks:
* By setting config. disable_env_checking=True
* by setting the env's `_skip_env_checking` property to True.

Reply:
I don't want to expose the _skip_env_checking attribute to users, so I'm going to just give a warning about the disable_env_checking flag in the config.

Reply:
done.